MAUI/Carousel-View/UIVirtualization.md

```diff
@@ -0,0 +1,138 @@
+---
+layout : post
+title: UIVirtualization in .NET MAUI Carousel View control | Syncfusion
+description: Learn here all about UIVirtualization support in Syncfusion .NET MAUI Carousel View (SfCarousel) control and more.
+platform : maui
+control : Carousel
+documentation : ug
+---
+
+# UIVirtualization in .NET MAUI Carousel View (SfCarousel)
+
+In UI virtualization concept, only the number of items that can be adaptable to the viewport of our device are arranged. Even, if the numerous items have been added to the collection, it loads only the viewport adaptable count of the carousel Items. Items are added at the right of the view when swiping the countable items in forward direction. At the same time, same number of items are removed at the left of the view for maintaining the same viewport items count. Similarly, items are added at the left of the view when swiping in backward direction for maintaining the same viewport items count. At the time, the same number of items are removed at the right of the view. Using this mechanism, virtualization concept is achieved in the carousel control. 
```

Comment:
In the UI virtualization concept, only the number of items that can be adaptable to the viewport of our device are arranged. Even, if numerous items have been added to the collection, it only loads the viewport adaptable count of the carousel items. Items are added at the right of the view when swiping the countable items in the forward direction. At the same time, the same number of items are removed at the left of the view to maintain the same viewport items count. Similarly, items are added at the left of the view when swiping backward to maintain the same viewport items count. At the same time, the same number of items are removed at the right of the view. Using this mechanism, the virtualization concept is achieved in the carousel control.

Reply:
Addressed the changes.